gateway/tests/test_scim.py

```diff
@@ -14,6 +15,10 @@
 from testcontainers.postgres import PostgresContainer
 from app_factory import create_app
 from models import OnboardingAppKey
+# pylint: disable-next=unused-import
+from scim_fdo import FDOExtension
+# pylint: disable-next=unused-import
+from scim_ethermab import EtherMABExtension
```

Comment:
This is similar to app.py. Why is this required and can we avoid it?

Reply:
Well, I can't avoid it but if you know of a way, great ;-)